src/blocks/sound.rs

```diff
@@ -174,16 +185,39 @@ pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
         )?),
     };
 
+    let mappings = config.mappings.map(|m| {
+        if config.mappings_use_regex {
+            Mappings::Regex(
+                m.into_iter()
+                    .filter_map(|(key, val)| Some((Regex::new(&key).ok()?, val)))
+                    .collect(),
```

Comment:
This silently skips invalid regexes. Is it intentional? (you can `.collect()` an iterator of results into `Result`)

Reply:
Actually it was intentional. My reasoning was that should we crash the whole thing just becuase one of the regex was incorrect?
And even if we get a result back, are we allowed to just `unwrap()` it? Otherwise the whole mapping will have to be skipped which is worse in my opinion.

Reply:
> should we crash the whole thing just becuase one of the regex was incorrect?

Not the whole thing, only this block.

> are we allowed to just unwrap() it?

No. Return from the block with this error.

I think it's better to let users know they have an error in their config rather than silently skip them. I imagine it being annoying to debug...